src/js/operations/Geo.js

```diff
@@ -0,0 +1,33 @@
+L.DNC.Geo = L.Class.extend({
```

Comment:
That's funny that you say that, because that's my same feeling so I generalized the file name. I'm fine continuing it as Turf, but I figured at some point we'll have turf operations interacting with other geo operations so going one big Geo file was my approach.

We're both in agreement, just different file naming ideas :grin: